CoffeeChat/Controller/GroupsViewController.swift

```diff
@@ -91,17 +96,24 @@ class GroupsViewController: UIViewController {
     }
 
     private func setupConstraints() {
+        let backButtonSize = CGSize(width: 10, height: 18)
         let backSize = CGSize(width: 86, height: 20)
-        let nextBackPadding: CGFloat = LayoutHelper.shared.getCustomVerticalPadding(size: 49)
-        let nextBottomPadding: CGFloat = LayoutHelper.shared.getCustomVerticalPadding(size: 90)
+        let skipBottomPadding: CGFloat = Constants.Onboarding.skipBottomPadding
+        let nextBottomPadding: CGFloat = Constants.Onboarding.nextBottomPadding
```

Comment:
maybe just use `Constants.Onboarding.skip/nextBottomPadding` in the constraints directly? agree w/ cameron that this is a lot of constants to declare upfront, so this could help. otherwise, i'd say if they're only used once it's okay to hardcode them directly (but that being said, i'd try to stick to one convention throughout the codebase)